src/components/PrimaryNav/PrimaryNav.js

```diff
@@ -212,11 +251,24 @@ const PrimaryNav = () => {
             </a>
           </li>
           <li className="p-list__item">
-            <span className="version">Version 0.0.2</span>
-            <span className="p-label--new">Alpha</span>
+            <span className="version">Version 0.1.0</span>
+            <span className="p-label--new">Beta</span>
           </li>
         </ul>
       </div>
+      {!isJuju ? (
+        <div className="p-primary-nav__bottom">
+          <li className="p-list__item">
```

Comment:
Missing `ul`. I think you just need a `p` here

Reply:
I need to keep the `li` for the same styling, but I added a wrapper `ul`